packages/@lightningjs/ui-components/src/components/Slider/Slider.js

```diff
@@ -323,9 +324,10 @@ export default class Slider extends Base {
   }
 
   get announce() {
-    return (
-      this._announce || (this.value !== undefined && this.value.toString())
-    );
+    if (this._announce !== undefined && this._announce !== null) {
+      return this._announce;
+    }
+    return this.value !== undefined && this.value.toString();
```

Comment:
Since the if statement before this line already checks `this._announce !== undefined`, that check can be removed from the check in this last return statement.
```suggestion
    return this.value.toString();
```